packages/react-radio/package.json

```diff
@@ -1,8 +1,7 @@
 {
   "name": "@fluentui/react-radio",
   "version": "9.0.0-beta.1",
```

Comment:
Should we change this version to 9.0.0-rc as well?

Reply:
No, if it's unstable it's still beta.